docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -0,0 +1,363 @@
+$ minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+```
+
+### Deploy the operator
+
+#### create the namespace
+
+```console
+$ kubectl create namespace send-receive-project
+$ kubectl config set-context --current --namespace=send-receive-project
+```
+
+Go to the root of the operator repo and install it:
+
+```console
+$ ./deploy/install_opr.sh
+```
+
+Wait for the Operator to start (status: `running`).
+```console
+$ kubectl get pod --namespace send-receive-project --watch
+```
+
+### Cert manager
+
+#### Install cert manager
+
+```console
+$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.5/cert-manager.yaml
+```
+
+#### Create the root issuer
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: send-receive-root-issuer
+  namespace: send-receive-project
+spec:
+  selfSigned: {}
+EOF
+```
+
+```console
+$ kubectl get issuers  -n send-receive-project -o wide send-receive-root-issuer --watch
+NAME                       READY   STATUS   AGE
+send-receive-root-issuer   True             45s
+```
+
+#### Create the issuer certificate
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: send-receive-issuer-cert
+  namespace: send-receive-project
+spec:
+  isCA: true
+  commonName: ArtemiseCloud send-receive issuer
+  dnsNames:
+    - issuer.artemiscloud.io
+  secretName: send-receive-issuer-cert-secret
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    name: send-receive-root-issuer
+    kind: Issuer
+EOF
+```
+
+```console
+kubectl get certificates  -n send-receive-project --watch
+NAME                       READY   SECRET                            AGE
+send-receive-issuer-cert   True    send-receive-issuer-cert-secret   24s
+```
+
+
+#### Create the issuer
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: send-receive-issuer
+  namespace: send-receive-project
+spec:
+  ca:
+    secretName: send-receive-issuer-cert-secret
+EOF
+```
+
+```console
+$ kubectl get issuer  -n send-receive-project --watch
+NAME                       READY   AGE
+send-receive-issuer        True    10s
+send-receive-root-issuer   True    50s
+```
+
+### Deploy the ActiveMQ Artemis Broker
+
+#### Ingress and ssl configuration
+
+The acceptor is configured as follows:
+* `port: 62626`
+* `sslEnabled: true` activates ssl
+* `expose: true` and `exposeMode: ingress` trigger an ingress creation
+* `sslSecret: send-receive-sslacceptor-0-svc-ing-ptls` gives the name of the
+  sercret store that'll be used to secure the connection
+* `ingressHost: send-receive-ss-0.send-receive-hdls-svc.send-receive-project.svc.cluster.local` gives the name of the endpoint, but we don't why we need it? TODO
```

Comment:
```suggestion
* `ingressHost: send-receive-ss-0.send-receive-hdls-svc.send-receive-project.svc.cluster.local` gives the name of the endpoint, but we don't why we need it? TODO
```
Could you clarify this question?

Reply:
Ah sorry I've left that :sweat_smile:

Reply:
We were wondering why there are duplicated fields with @gtully and I have left some draft of the question in the middle of it.